sale_input_barcode_gs1/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module builds on sale_input_barcode, making use of the full barcode information given by a barcode/datamatrix device.
+
+This module may also create lot and/or select lot if used in combination with sale_order_lot_selection
```

Comment:
OCA icon changed, I thought ?

Reply:
I assume you're referring to the file `icon.png`? I can't find an instance of an OCA module that has the new logo. I think it's used in the Readme files, but not for the modules? I may be wrong.